pd/src/app.rs

```diff
@@ -335,29 +333,106 @@ impl App {
                 .unwrap()
                 .add_transaction(verified_transaction);
 
-            increment_counter!("node_transactions_total");
+            metrics::increment_counter!("node_transactions_total");
```

Comment:
Previously this was just `increment_counter!`; for tracing and metrics calls I think it's useful not to import the specific functions but just to use the crate name, so that it's immediately clear that it's some instrumentation rather than functional behavior.